README.md

```diff
@@ -40,13 +40,19 @@ private_movie_views = \
 
 # Calculate the private sum of ratings per movie
 dp_result = private_movie_views.sum(
-    SumParams(max_partitions_contributed=2,
-              max_contributions_per_partition=2,
+    SumParams(
+              # Limits to how much one user can contribute:
+              # .. at most two movies rated per user
```

Comment:
it might be not obvious what if the user has more contributions. Maybe we can add something, that if more, then sampling is performed ?

Reply:
Yep, adding a comment about that.